bats/fb-destroy-organization.bats

```diff
@@ -12,5 +12,5 @@ setup() {
 }
 
 @test "Delete an Organization" {
-  hammer organization delete --name="${ORGANIZATION}"
+  hammer organization delete --name="Empty Organization"
```

Comment:
was this intentional?

Reply:
yes, at one point I recall that it was erroring trying to delete "Test Organization" so I changed the deleted org to "Empty Organization." Figured it didn't matter much.

Reply:
The Test Organization has all kinds of entities created in it, so by deleting that org we can ensure all of those entity deletions succeed. Deleting the Empty Org isn't as valuable of a test since it barely does anything except delete the Organization record + candlepin Owner record

Reply:
ok, let me try running the pipeline again and then maybe I'll remember more about what it broke :wink:

Reply:
It passed again, no errors.  I guess that was just a transient or unrelated gremlin. I think it should be good to go now!